### DISCUSSION TITLE
question about skin editor and built in tablet drivers and UI and delay

### DISCUSSION THREAD

**Discussion by bogdic00kie** [ppy/osu#21853] [Q&A] [answered]
idk under witch category to submit this so im sorry if its in the wrong one

1 .will there be a way to edit somethin in all the skins at once so you don't have to edit all of them manually or to reset stuff to default? and some skins are kinda broken in lazer (my example is 404 skinnotfound )

2. also i personally prefer the devocub tablet drivers because i also use my tablet to draw and it has filters and antichatter for my horrible shake and tablet doesn't work or works really weird if i don't use the inbuilt drivers witch i find a bit limiting, will compatibility with other tablet drivers be fixed?

3. will there be UI/menu presets/styles? like make the menu look a bit more like stable or smth like that? maybe some that can be unlocked through doing certain things?

4. on osu stable my delay is like 2.3 - 3 or smth like that and on osu lazer its 4 - 5 (idk if its my settings or not) and the game feels a bit "different" not necessarily delayed or laggy but feels a bit off compared to stable idk if its just me or not  

sorry if i put this under the wrong category or phrased things wrong

**Comment by peppy** [comment, marked as answer]
1. please attach your broken skin
2. you can disable built-in tablet support in settings. un-check the check box.
3. we have UI updates coming which should make you feel more at-home.
4. i wouldn't focus on the numbers too much. they aren't 1:1 comparable.

**Reply by bogdic00kie** [reply]
im extremely sorry for the late response 

i cant find the skins folder in lazer and when i tried attaching the exported skin it didnt let me due to it being in an unsupported format
the issue only happens in skins that have the accuracy number and pie thing in the top middle rather than the top right like most skins

if i turn off the osu lazer driver the tablet only works if i hover really low cause if i drag the cursor gets stuck and if i hover higher the cursor moves very choppy and weird almost like it teleports. the osu lazer tablet driver feels a bit weird maybe due to it not having antichatter or filters and/or me not playing for a long time or maybe because most people use open tablet driver(OTD) and i use devocub witch is older and might not be as compatible  

i also find the skin editor weird to use and if i accidentally move something or delete it i cant really undo/reset the skin to default

i find it a bit annoying to change keybinds due to not being used to the new ui but thats probably me being smooth brained although i wish that for some settings like sensitivity we could input the number instead of having to drag a slider but thats just me

other than that i did notice that from the last time i played lazer it runs really well, without drops or stutters so 
i really did notice that the game did get better than last time i checked and im stoked to see how it will be in the future

**Reply by peppy** [reply]
That's because your skin is designed in an unsupported way. I'm not sure we can support this going forward, so you might need to manually fix it in the layout editor.
